### PR TITLE
MOTECH-2586 Fixed the bug related to FieldSettings

### PR DESCRIPTION
**Pull request by shanmukhm**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by pgesek** [comment]
test this please

File: platform/mds/mds-web/src/main/resources/webapp/js/directives.js [hunk 1]
```diff
@@ -2268,9 +2268,7 @@
                         fieldPath = fieldPath.substring(fieldPath.indexOf('.') + 1);
                     }
 
-                    value = _.isBoolean(ngModel.$modelValue)
-                        ? !ngModel.$modelValue
-                        : ngModel.$modelValue;
+                    value = ngModel.$modelValue;
```
Comment (pgesek) [inline platform/mds/mds-web/src/main/resources/webapp/js/directives.js]: @shanmukhm was there an issue only with boolean fields? The ticket does not mention that

Comment (shanmukhm) [inline platform/mds/mds-web/src/main/resources/webapp/js/directives.js]: @pgesek I checked it again and found that not only settings but also validations are not being saved (not only boolean values). So, I'll update the description and also rework on this ticket.

**Comment by pgesek** [comment]
test this please

**Comment by pgesek** [comment]
test this please

**Comment by pgesek** [comment]
test this please

**Pull request merged**